code/modules/spacepods/spacepod.dm

```diff
@@ -589,11 +589,21 @@ obj/spacepod/proc/add_equipment(mob/user, var/obj/item/device/spacepod_equipment
 			desc = "A rough looking space pod meant for industrial work"
 	update_icons()
 
+/obj/spacepod/proc/can_use_verb()
+	if(usr.incapacitated())
+		to_chat(usr, "<span class='notice'>You cannot do this while unconscious or restrained.</span>")
+		return FALSE
+	return TRUE
+
 /obj/spacepod/verb/toggle_internal_tank()
 	set name = "Toggle internal airtank usage"
 	set category = "Spacepod"
 	set src = usr.loc
 	set popup_menu = 0
+
+	if(!can_use_verb())
```

Comment:
That's what I had originally, but I noticed other places in the code gave you an error message if an action failed because of incapacitation. I can change it back if no error message is preferable.